src/os/bluestore/BlueStore.cc

```diff
@@ -13433,7 +13432,8 @@ int BlueStore::_do_alloc_write(
 
       // FIXME: memory alignment here is bad
       bufferlist t;
-      int r = c->compress(wi.bl, t);
+      int32_t compressor_message;
```

Comment:
Please init with zero to avoid arbitrary garbage in this field for all other compressors.

Reply:
sorry,it's my fault